web/services/hosts_next.go

```diff
@@ -59,9 +66,8 @@ func (s *hostsNextService) GetAll(filters map[string][]string) (models.HostList,
 		host := h.ToModel()
 		host.Health = computeHealth(&h)
 
-		health, ok := filters["health"]
-		if ok && len(health) > 0 {
-			if !internal.Contains(health, host.Health) {
+		if filter != nil && len(filter.Health) > 0 {
+			if !internal.Contains(filter.Health, host.Health) {
```

Comment:
I know what you'll say, but making a function for these conditions is always viable to me :D

Reply:
yep im not sure this will stay i will keep on the radar for the next iterations!